deployment/hasura/migrations/AerieMerlin/0_create_schema_migrations/up.sql

```diff
@@ -0,0 +1,33 @@
+create schema migrations;
+create table migrations.schema_migrations (
```

Comment:
Something I noticed when I applied the migrations is that Hasura doesn't automatically track the new tables. In the event we want it to (for example, in order to create a new relationship), do we know if there's a way to do this or will the user just have to copy in the latest schema's metadata and refresh it after they finish migrating?

Reply:
If we want Hasura to track a table, I think we can just add it to `deployment/hasura/metadata/databases/*/tables/`. The tables exposed to Hasura should only be those meant to be exposed to downstream clients and exported through the Aerie API. I'm not sure that's true of `schema_migrations`.

Reply:
(I think that once the `metadata/` is altered, customers need to reload the metadata through either the terminal or the Hasura frontend; I don't think it will autoload any changes. But it would be a one-step change as part of an upgrade in that case, and not something they need to go in and directly configure themselves.)

Reply:
> will the user just have to copy in the latest schema's metadata and refresh it after they finish migrating?
Yeah - I think this is what will end up happening :+1: Thanks for bringing that up - there are no corresponding "metadata" migrations - so the metadata will need to be updated wholesale

Reply:
(Oops, sorry -- I was distracted and didn't realize I didn't completely read @Mythicaeda's full message. Sorry for missing the point 🤦‍♂️)

Reply:
Huh. I know that when applying a migration via the Hasura CLI, it reloads the metadata, which to me implied that it might've been changeable alongside migrations, but reading the docs, I think it is just something that a user will need to replace and reload.

Also, looking at [the docs](https://hasura.io/docs/latest/migrations-metadata-seeds/migrations-metadata-setup/), there's apparently a way to mark a migration as applied without executing it:
```
# mark the squashed migration as applied on this server
hasura migrate apply --version "<squash-migration-version-timestamp-number>" --skip-execution --database-name <database-name>
```

Which should be useful for at least the aerie-helper migration script.

(And, yeah, I agree, `schema_migrations` probably doesn't need to be exposed and tracked, but in general if we add a table, we'll _probably_ want to track/expose it.)

Reply:
Agh, I submitted that right before I got the email for your replies. Thanks for the clarification @mattdailis!